app.rb

```diff
@@ -0,0 +1,174 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './book'
+require_relative './rental'
+require 'date'
+
+class App
+  def initialize
+    @people = []
+    @book = []
+    @rentals = []
+  end
+
+  def run
+    puts 'Welcome to school library App!'
+    puts ''
+    puts ''
+    puts 'Please put an option by entering a number!'
```

Comment:
- [optional] you can as well add which output it should give if user selects and option which is not found on the list. nice work

Reply:
Thank you @tufoinnkuo10 for your review and suggestion.

Reply:
> Thank you @tufoinnkuo10 for your review and suggestion. i however provided for the scenerio in [these lines ](https://github.com/Efiamotu-1/OOP-School-Library/blob/ec1e4fb3d696ac44b04351dd8fa3e7c571d7c769/app.rb#L53-L55)